### ISSUE TITLE
AWS Tools for Windows PowerShell 4.1.3.0 disappeared

### ISSUE THREAD

**Issue by ganeshnj** [aws/aws-tools-for-powershell#182]
## Description 
We released AWS Tools for Windows PowerShell 4.1.3.0  on Thu 29-Oct-2020. At the same time PowerShell Gallery had an [issue](https://github.com/PowerShell/PowerShellGallery/issues/135) and mitigate the issue, they [migrated the database](https://github.com/PowerShell/PowerShellGallery/blob/master/psgallery_status.md) to a previous date which appears to have removed version 4.1.3.0.

We have created [an issue](https://github.com/PowerShell/PowerShellGallery/issues/143) to Microsoft for investigation.

### Logs
```
 Failed to publish module 'AWSPowerShell': 'dotnet cli failed to nuget push Pushing AWSPowerShell.4.1.3.nupkg to 'https://www.powershellgallery.com/api/v2/package/'...
    PUT https://www.powershellgallery.com/api/v2/package/
    InternalServerError https://www.powershellgallery.com/api/v2/package/ 12094ms
    PUT https://www.powershellgallery.com/api/v2/package/
    InternalServerError https://www.powershellgallery.com/api/v2/package/ 4969ms
    PUT https://www.powershellgallery.com/api/v2/package/
    InternalServerError https://www.powershellgallery.com/api/v2/package/ 6829ms
EXEC : error : Response status code does not indicate success: 500 (There is a conflict with the name and version of your package and another package. Please change your package's name or version and try again.).
```

## Resolution
Pending

---

This is a :bug: bug-report

**Comment by boblodgett** [comment]
To work around the issue we have released AWS Tools for PowerShell 4.1.4.0.

**Comment by ganeshnj** [comment]
Given we don't have customer asking for 4.1.3.0, we can close this issue.
